justfile

```diff
@@ -32,11 +33,18 @@ run-services:
   # if someone has a better solution, be my guest
   sed 's/GATEWAY_IPV4/'` ip addr show eth0 | grep inet | awk '{print $2}' | cut -d/ -f1 | head -n1`'/g' \
     docker/docker-compose.monkey-patch-linux.yml | tee tmp/docker-compose.linux.yml
-  cd docker && docker compose -f docker-compose.yml -f ../tmp/docker-compose.linux.yml up 
 
 clean-services:
   cd docker && docker compose down -v
 
+tor:
+  docker run --rm -ti \
+    -p 5000:5000 \
```

Comment:
socator port could be set via .env

Reply:
> socator port could be set via .env

We can't set it, `socator` has `5000` (see hardcoded https://github.com/beacloudgenius/socator/blob/main/tor-socat.sh#L25)

Reply:
only for the docker internal port is hardcoded, I believe. Thats fine. The forwarded port on the host should be able to change because some environments might already use something on localhost:5000

Reply:
Ok, updated in e1c0605